casual.html

```diff
@@ -96,4 +110,5 @@ <h1>The Casual Guide will be available soon! In the meantime, the <a href="https
 	}).call(window);
 </script>
 <script src="collapsible.js"></script>
+<script src="dragging.js"></script>
```

Comment:
is it possible to have this script in the HEAD?

Reply:
I've tried putting window.onload calls at the start of dragging.js and collapsible.js so they can go in the header but I can't seem to get both of these functions to work properly. From searching online it appears using multiple onload functions on the same page can cause problems so fixing this will be more involved than I thought. I'll fix this after other fixes that more directly impact the user.

Reply:
yeah I think this script can't be in the HEAD without changes to it. it is fine where it is.